setup.py

```diff
@@ -38,6 +38,7 @@
 
 install_requests_requires = [
     "requests>=2.23,<3",
+    "requests_toolbelt>=0.9.1",
```

Comment:
Please add `,<1` here so that a new major release of requests_toolbelt will not break gql.

Reply:
done